lms/static/certificates/sass/_layouts.scss

```diff
@@ -0,0 +1,324 @@
+// ------------------------------
```

Comment:
NOTE: this partial controls almost all of the layout controls, including responsive details.